Source/WTF/wtf/fast_float/bigint.h

```diff
@@ -116,10 +116,7 @@ struct stackvec {
   // appended item.
   void resize_unchecked(size_t new_len, limb value) noexcept {
     if (new_len > len()) {
-      size_t count = new_len - len();
-      limb* first = data + len();
-      limb* last = first + count;
-      ::std::fill(first, last, value);
+      ::std::fill(data + len(), data + new_len, value);
```

Comment:
This is third party imported file. Do not touch directly.

Reply:
Nowhere is it mentioned that this is the case though?